### ISSUE TITLE
Add Plugin Release Dates

### ISSUE THREAD

**Issue by nhmathis** [UrbanCode/IBM-UCx-PLUGIN-DOCS-OLD#19]
The original plugin website had a way to see _when_ a version was released. This was helpful in understanding how recent the plugin it is and general debug understanding.
Can we add those release dates back? I recognize this is likely a slow process.
Example
![image](https://user-images.githubusercontent.com/14111822/179226258-27e574ab-1c31-4f74-ba59-91e9ce971f8d.png)

**Comment by osmanburucu** [comment]
I do not think that this could be possible for the old versions as the timestamp information got lost by copieng them around. But newer versions should be possible. This is in the hands of the plugin providers to add it to the document
